src/privateai_client/tests/test_request_objects.py

```diff
@@ -927,3 +928,26 @@ def test_bleep_request_to_dict():
     bleep_request = BleepRequest(file=file, timestamps=timestamps).to_dict()
     assert bleep_request["file"]["data"] == "test"
     assert bleep_request["timestamps"][0]["start"] == 0
+
+
+def test_synthetic_text():
+    processed_text = ProcessedText(
+        type="SYNTHETIC",
+        synthetic_entity_accuracy="standard",
+        preserve_relationships=True,
+    )
+    text_request = request_objects.process_text_obj(
```

Comment:
```suggestion
    text_request = ProcessTextRequest(
```
just to avoid the extra import :)